.github/workflows/build.yml

```diff
@@ -0,0 +1,45 @@
+name: build
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        BUILD_TYPE: ["Debug"]
+        os: [ windows-latest ]
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Get MSVC
+      uses: ilammy/msvc-dev-cmd@v1
+
+    - name: Get specific version CMake, v3.21.2
+      uses: lukka/get-cmake@v3.21.2
+
+    - name: Install rust stable
+      uses: actions-rs/toolchain@v1
+      with:
+        toolchain: stable
+
+    # need to use the latest winrt in sdk, i.e. pass the version to cmake
+    - name: find latest win sdk
```

Comment:
This is not needed.

Reply:
I think this is for the requirement "service fabric runtime installation"

Reply:
link: https://learn.microsoft.com/en-us/azure/service-fabric/service-fabric-get-started

Reply:
Sdk is installed in action machines by default. There is no need to query the version.
The original code was need to configure cpp headers.

Reply:
Removed and it works well. Thanks youyuan